randovania/games/am2r/logic_database/header.txt

```diff
@@ -228,7 +228,7 @@ Templates
               Missiles ≥ 30 or Super Missiles ≥ 6
               All of the following:
                   # Missile-Less Requirements
-                  Charge Beam
+                  Charge Beam and Underwater Power Grip Wall
```

Comment:
Personally I would rather this be on the connection to the dual gamma's for readability and leave this as fight requirements but I won't block the PR over it.

Reply:
IMO it fits nicer here, because you have to use those abilities during the combat.